edx_proctoring/views.py

```diff
@@ -1364,7 +1365,8 @@ def post(self, request):
         exam_attempt_id = create_exam_attempt(
             exam_id=exam_id,
             user_id=user_id,
-            taking_as_proctored=attempt_proctored
+            taking_as_proctored=attempt_proctored,
+            is_verified_name_enabled=waffle.switch_is_active(VERIFIED_NAME_FLAG)
```

Comment:
I think this should be `is_enabled`, not `switch_is_active` because this is defined as a WaffleFlag in name affirmation. I'm also not positive about how flags can be accessed across services, have you tested that this works locally?

Reply:
Could be very off base with the function name, I just assumed it would be closer to `VERIFIED_NAME_FLAG.is_enabled()` which is in the name affirmation repo.

Reply:
I was just patterning off other waffle calls in proctoring, I wasn't aware there was a whole set of possible alternate calls. I'll look into it more